beams/typing_helper.py

```diff
@@ -0,0 +1,45 @@
+"""
+typing helper:
+Define commonly used "aggregator" types, lagerly custom function handles
+Hint at rational behind function handles as entrypoint for debugging / learning
+"""
+
+
+from typing import Callable, Any
+from multiprocessing import Queue, Event, Value
+
+import py_trees
+
+from beams.behavior_tree.VolatileStatus import VolatileStatus
+
+
+'''
+Evaluatable: function handle that returns a boolean , accpeting arbitrary arugments 
+'''
+Evaluatable = Callable[[Any], bool]
```

Comment:
https://docs.python.org/3/library/typing.html#annotating-callable-objects

```suggestion
Evaluatable = Callable[..., bool]
```